src/apps/loyalty/server/middlewares/user.ts

```diff
@@ -0,0 +1,21 @@
+import { NextFunction, Request, Response } from "express"
+import { fetchCollectorProfile } from "../gravity"
+
+export default (req: Request, res: Response, next: NextFunction) => {
```

Comment:
This approach sort of gives me pause.

For one, we're using Backbone by attaching it to the user. We actually have https://github.com/artsy/artsy-passport/issues/104 and may hopefully be able to remove Backbone entirely very soon. Is there a way to just use plain JS objects? (and is `res.locals` the place of choice for it?)

Also, doing it this way loses the type information we have associated with the response object. From https://github.com/artsy/reaction-force/blob/2a364e2ebe1ed6d8c0eec63add9bfc41927d9a4b/src/apps/loyalty/server/enroll_loyalty_applicant.ts#L20, the return value is typed, but doing it this way loses that and we just get an `any`.